core/src/plugins/filed/python/libcloud/BareosLibcloudApi.py

```diff
@@ -50,6 +50,15 @@ def probe_driver(options):
             return "success"
         return "failed"
 
+    @staticmethod
+    def redirect_stdout_stderr():
+        if sys.stdout != None:
+            sys.stdout.close()
+        if sys.stderr != None:
+            sys.stderr.close()
+        sys.stdout = open("/dev/null", "w")
+        sys.stderr = open("/dev/null", "w")
```

Comment:
you could have reused the filehandle you opened for sys.stdout and use it for sys.stderr, too.
However, two filedescriptors won't hurt either.